src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlServerSortingValidator.cs

```diff
@@ -10,14 +10,16 @@
 using Microsoft.Health.Fhir.Core.Features;
 using Microsoft.Health.Fhir.Core.Features.Search;
 using Microsoft.Health.Fhir.Core.Models;
+using Microsoft.Health.Fhir.ValueSets;
 
 namespace Microsoft.Health.Fhir.SqlServer.Features.Search
 {
     internal class SqlServerSortingValidator : ISortingValidator
     {
-        private static readonly HashSet<string> _supportedParameterNames = new HashSet<string>(StringComparer.Ordinal)
+        private readonly HashSet<SearchParamType> _supportedSortTypes = new HashSet<SearchParamType>()
```

Comment:
This is in main and will need updating too: https://github.com/microsoft/fhir-server/blob/main/src/Microsoft.Health.Fhir.SqlServer/Features/Storage/Registry/SqlServerSearchParameterStatusDataStore.cs#L96